src/extension.ts

```diff
@@ -80,6 +80,14 @@ export function activate(context: vscode.ExtensionContext) {
                 vscode.window.showInformationMessage('Stack Protocol Server is starting, please try again later!');
             }
         });
+        
+
+        context.subscriptions.push(
+            // user commands
+            vscode.commands.registerCommand('adapter.restartserver', () => {
```

Comment:
Should get `context` as a parameter the same way other server related commands do.